cmd/renterd/config.go

```diff
@@ -241,6 +243,7 @@ func parseCLIFlags() {
 	flag.Uint64Var(&cfg.Worker.UploadMaxMemory, "worker.uploadMaxMemory", cfg.Worker.UploadMaxMemory, "Max amount of RAM the worker allocates for slabs when uploading (overrides with RENTERD_WORKER_UPLOAD_MAX_MEMORY)")
 	flag.Uint64Var(&cfg.Worker.UploadMaxOverdrive, "worker.uploadMaxOverdrive", cfg.Worker.UploadMaxOverdrive, "Max overdrive workers for uploads")
 	flag.DurationVar(&cfg.Worker.UploadOverdriveTimeout, "worker.uploadOverdriveTimeout", cfg.Worker.UploadOverdriveTimeout, "Timeout for overdriving slab uploads")
+	flag.DurationVar(&cfg.Worker.UploadSectorTimeout, "worker.uploadSectorTimeout", cfg.Worker.UploadSectorTimeout, "Timeout for upload of individual sectors of a slab")
```

Comment:
These flags and the config values will need to be added to the README as well for other people to find them.